ddtrace/appsec/processor.py

```diff
@@ -86,15 +86,21 @@ class _Addresses(object):
     "x-real-ip",
 }
 
-_COLLECTED_HEADER_PREFIX = "http.request.headers."
+
+_COLLECTED_RESPONSE_HEADERS = {
+    "content-length",
+    "content-type",
+    "Content-Encoding",
+    "Content-Language",
+}
 
 
-def _set_headers(span, headers):
-    # type: (Span, Dict[str, Union[str, List[str]]]) -> None
+def _set_headers(span, kind, to_collect, headers):
```

Comment:
If we expect `to_collect` to essentially default to `_COLLECTED_RESPONSE_HEADERS`, I would move the global into a local for a slightly faster look up, and also make it a `frozenset`. WDYT?

Reply:
indeed, `_COLLECTED_RESPONSE_HEADERS` is a subset of `_COLLECTED_REQUEST_HEADERS`. But We call the method with both for each attack, would there still be a gain?
+1 for `frozenset`